src/dto/notes/update-note.dto.ts

```diff
@@ -0,0 +1,33 @@
+import { Transform, TransformFnParams } from 'class-transformer';
+import { Length, IsOptional, IsUUID, IsDate, MinDate } from 'class-validator';
+
+import {
+  MIN_DATE_VALUE,
+  TASK_MAX_LENGTH,
+  TASK_MIN_LENGTH,
+  UUID_LENGTH,
+} from 'common';
+
+export class UpdateNoteDto {
```

Comment:
do we need to duplicate it? or we can use extends here with needed extra fields?